pkg/env/sensor.go

```diff
@@ -21,7 +21,7 @@ var (
 	ScannerSlimGRPCEndpoint = RegisterSetting("ROX_SCANNER_GRPC_ENDPOINT", WithDefault("scanner.stackrox.svc:8443"))
 
 	// ScannerV4GRPCEndpoint is used to communicate with the Scanner V4 endpoint in the same cluster.
-	ScannerV4GRPCEndpoint = RegisterSetting("ROX_SCANNER_V4_GRPC_ENDPOINT", WithDefault("scanner-v4.stackrox.svc:8443"))
+	ScannerV4GRPCEndpoint = RegisterSetting("ROX_SCANNER_V4_GRPC_ENDPOINT", WithDefault("scanner-v4-indexer.stackrox.svc:8443"))
```

Comment:
wonder if we should consider making it clear this is the matcher in the var and env var names

Reply:
Would be the `indexer` in this case :)

That would be a slightly bigger change as have to modify the helm charts (and potentially the operator), because PRs for both helm and operator are in flight modifying I don't want to add an unnecessary merge conflict - created ROX-21709 to address this in the future.